Sources/CodeEditTextView/ItemBox/ItemBoxWindowController.swift

```diff
@@ -0,0 +1,240 @@
+//
+//  TextView+ItemBox.swift
+//  CodeEditTextView
+//
+//  Created by Abe Malla on 6/18/24.
+//
+
+import AppKit
+import LanguageServerProtocol
+
+/// Represents an item that can be displayed in the ItemBox
+public protocol ItemBoxEntry {
+    var view: NSView { get }
+}
+
+/// Padding at top and bottom of the window
+let WINDOW_PADDING: CGFloat = 5
```

Comment:
We're a bit inconsistent with this. Here you declared the constant with capital letters and in the extension you have constants declared normally, within the class and in lowercase.